README.md

```diff
@@ -240,7 +243,7 @@ CREATE TABLE STAGING
 `csv2db` will use `VARCHAR(1000)` as default data type for all columns for the staging table. If you wish to use a different data type, you can specify it via the `-c` option:
 
 ```sql
-$ ./csv2db generate -f test/resources/201811-citibike-tripdata.csv -t STAGING -c CLOB
+$ ./csv2db generate -f test/resources/201811-citibike-tripdata.csv -t STAGING -c CLOB -o mysql
```

Comment:
I don't think this should change in the ReadMe. For the majority of use cases, running without the database type should be sufficient to generate the table DDL

Reply:
Thats fair. Sorry for changing this. Fixed in https://github.com/csv2db/csv2db/pull/51/commits/5a890c37c8575a6b6d95bc3f418fe8ef682e672f.